MapboxDirections/Attributes/MBSpeedLimit.swift

```diff
@@ -1,5 +1,10 @@
 import Foundation
 
+/**
+ Speed value for a segment which is unknown.
+ */
+public let MBSpeedIsInvalid: Double = -1.0
```

Comment:
The suggestion in https://github.com/mapbox/MapboxDirections.swift/pull/250#discussion_r176269902 was to add a constant of type `MBSpeedLimit`, not a constant for the magic number alone. But we can only make it a constant if we make `MBSpeedLimit` a struct. Since we can’t, then we’ll need to create a class factory method on SpeedLimit instead:

```swift
class var invalid = SpeedLimit(value: -1, unit: .kilometersPerHour)
```